### PR TITLE
Clean up AIP-132: List

### PR DESCRIPTION
**Pull request by rofrankel**
Minimal changes - just fixing broken links and making the style match other recently adopted AEPs.

### CODE REVIEW

File: aep/general/0132/aep.md.j2 [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
 
 ### Requests
 
-Collection `GET` operations **must** be made by sending a `GET` request to the
-resource collection's URI:
+`List` operations **must** be made by sending a `GET` request to the resource
+collection's URI:
 
 ```http
 GET /v1/publishers/{publisher}/books HTTP/2
```
Comment (toumorokoshi) [inline aep/general/0132/aep.md.j2:26]: nit: dding "HTTP/2" here feels weird to me. I know it was from the previous version though.

File: aep/general/0132/aep.md.j2 [hunk 2]
```diff
@@ -126,8 +130,8 @@ change.
 
 ### Soft-deleted resources
 
-Some APIs need to "[soft delete][aip-135]" resources, marking them as deleted
-or pending deletion (and optionally purging them later).
+Some APIs need to "[soft-delete][]" resources, marking them as deleted or
```
Comment (toumorokoshi) [inline aep/general/0132/aep.md.j2:133]: as mentioned in the delete AEP: I think we should drop the pattern entirely and replace it.

But if it's too much trouble, I could do that in my soft delete proposal.

File: aep/general/0132/aep.md.j2 [hunk 3]
```diff
@@ -165,9 +169,9 @@ List operations also implement a common request message pattern:
 
 - A `parent` field **must** be included unless the resource being listed is a
   top-level resource. It **should** be called `parent`.
-  - The field **should** be [annotated as required][aip-203].
-  - The field **should** identify the [resource type][aip-123] of the resource
-    being listed.
+  - The field **should** be [annotated as required][aep-203].
```
Comment (toumorokoshi) [inline aep/general/0132/aep.md.j2:172]: why not must?

File: aep/general/0132/aep.md.j2 [hunk 4]
```diff
@@ -165,9 +169,9 @@ List operations also implement a common request message pattern:
...
+  - The field **should** identify the [resource type][] of the resource being
```
Comment (toumorokoshi) [inline aep/general/0132/aep.md.j2:173]: why not must?

**Review by toumorokoshi — CHANGES_REQUESTED** [review]
just a couple questions on things that should probably be **MUST**. Otherwise LGTM!

File: aep/general/0132/aep.md.j2 (hunk 1 above)
Comment (rofrankel) [reply aep/general/0132/aep.md.j2:26]: I agree.  We need a broader style pass on the standard method AEPs to get them stylistically consistent.  Are you okay with doing that in a separate PR?

File: aep/general/0132/aep.md.j2 (hunk 2 above)
Comment (rofrankel) [reply aep/general/0132/aep.md.j2:133]: I think we should leave this guidance in until we fully excise soft-delete...are you okay with removing this when we merge the alternative?

File: aep/general/0132/aep.md.j2 (hunk 3 above)
Comment (rofrankel) [reply aep/general/0132/aep.md.j2:172]: Done.

File: aep/general/0132/aep.md.j2 (hunk 4 above)
Comment (rofrankel) [reply aep/general/0132/aep.md.j2:173]: Done.

File: aep/general/0132/aep.md.j2 (hunk 1 above)
Comment (toumorokoshi) [reply aep/general/0132/aep.md.j2:26]: I'd rather it just be removed now. But alternatively if you can file an issue it's fine.

Comment (toumorokoshi) [reply aep/general/0132/aep.md.j2:26]: One nice thing is that if it's an issue, we could put it in the 'starter tasks' for the barn-raising.

File: aep/general/0132/aep.md.j2 (hunk 2 above)
Comment (toumorokoshi) [reply aep/general/0132/aep.md.j2:133]: yes.

**Review by toumorokoshi — APPROVED** [review]
LGTM! the outstanding comment can be resolved once an issue is filed.

File: aep/general/0132/aep.md.j2 (hunk 1 above)
Comment (rofrankel) [reply aep/general/0132/aep.md.j2:26]: Created an issue and tagged accordingly, thanks.

**Pull request merged**